puppet/zulip_ops/manifests/postgres_common.pp

```diff
@@ -6,20 +6,21 @@
                                  "pv",
                                  "python3-pip",
                                  "python-pip",
-                                 # "python3-gevent", # missing on trusty
-                                 "python-gevent",
                                  # Postgres Nagios check plugin
                                  "check-postgres",
                                  ]
   package { $internal_postgres_packages: ensure => "installed" }
 
+  exec {"pip2_ensure_latest":
+    command => "/usr/bin/pip2 install -U pip==9.0.1",
+    creates => "/usr/local/bin/pip2",
+    require => Package['python-pip'],
+  }
+
   exec {"pip_wal-e":
-    # On trusty, there is no python3-boto or python3-gevent package,
-    # so we keep our `wal-e` explicitly on Python 2 for now.
-    command  => "/usr/bin/pip2 install git+git://github.com/zbenjamin/wal-e.git#egg=wal-e",
+    command  => "/usr/local/bin/pip2 install 'boto==2.4.0' 'gevent==1.2.2' git+git://github.com/zbenjamin/wal-e.git#egg=wal-e",
     creates  => "/usr/local/bin/wal-e",
-    require  => Package['python-pip', 'python-boto', 'python-gevent',
-                        'lzop', 'pv'],
+    require  => Package['python-pip', 'lzop', 'pv'],
```

Comment:
Have you tested this in a production environment/

Reply:
Not yet; I rely too heavily on Travis for experimenting stuff. If this is not captured by the current test suite, then yeah, I'm blocked by the testing.

Reply:
Yeah, Travis doesn't test anything under `zulip_ops`.